servers/quarkus-server/src/main/java/com/dremio/nessie/server/config/ApplicationConfig.java

```diff
@@ -117,6 +117,9 @@ public VersionStoreDynamoConfig getVersionStoreDynamoConfig() {
     @ConfigProperty(name = "initialize", defaultValue = "false")
     boolean isDynamoInitialize();
 
+    @ConfigProperty(name = "setupTables", defaultValue = "false")
+    boolean isSetupTables();
```

Comment:
(style) the property name is not really correct english (but I guess same above). I'm not sure to understand what's the difference with the `initialize` property above...